client/src/components/Home/PartnersSection/PartnersSection.js

```diff
@@ -1,13 +1,42 @@
-import React from 'react';
-import 'bootstrap/dist/css/bootstrap.min.css';
-import './PartnersSection.css';
-
-function PartnersSection() {
-  return (
-    <div className="partners-section">
-      <h1>This is the Partners section.</h1>
-    </div>
-  );
-}
-
-export default PartnersSection;
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './PartnersSection.css';
+import {PartnersList} from './PartnersList.js';
+
+class PartnersSection extends React.Component {

```

Comment:
You can use functional component with hooks here instead of using a class component.

Reply:
SInce most of the time i prefer class component,i don't usually use hooks, i think i will need some help in implementing the following code using hooks 😅.

Reply:
ok, I will be making a commit to your fork!

Reply:
Thank you sir :-)